tasks/main.yml

```diff
@@ -47,6 +47,12 @@
   tags:
     - client_keys
 
+- name: generate revocation list and clean up
+  include: revocation.yml
+  when: certificate_revocation_list is defined
```

Comment:
`openvpn_revoke_these_certs` is clearer. Has the `openvpn_` prefix, since variables are global, and `revoke` as the second word since it's an action.